app/controllers/volunteer_ops_controller.rb

```diff
@@ -36,7 +36,7 @@ def create
     @volunteer_op = VolunteerOpForm.new(volunteer_op_params)
     result = @volunteer_op.save
     result ? vol_op_redirect(t('volunteer.create_success')) : render(:new)
-    UpdateSocialMedia.new.post @volunteer_op if result
+    UpdateSocialMedia.new.post @volunteer_op if result && ENV["RAILS_ENV"] != 'development'
```

Comment:
@tansaku Is there a better way to do this?

Reply:
@NickSchimek I think we might prefer a check like Rails.env.development?  but even better might be to have a global twitter_enabled feature ...

this is a related approach for emails:  https://stackoverflow.com/a/7729789/316729

maybe a better all round approach would be to have separate twitter keys in development and have an actually develop twitter account dedicated to receiving tweets ...

Reply:
A dedicated twitter account would probably be the best route to go, but would it cause any issues for someone developing off line? If they were to hit this controller offline in the development environment?

Reply:
that doesn't sound like a big issue - the tweet would just not go out - but in development mode I don't think we care ...

Reply:
@tansaku I tried running it offline and it threw a fit... I received a 401 unauthorized error in airbrake and it rendered the debug page. It would sure be nice if I could tweet on a separate thread... not sure if that's possible, anyways I suppose we could catch this error... But for now I updated to the more idiomatic version, pulled in latest changes from develop, fixed all conflicts, and it's once again ready to merge.